packages/horizon-householder-appeal-publish/src/publishDocuments.js

```diff
@@ -0,0 +1,51 @@
+const axios = require('axios');
+const config = require('./config');
+
+/**
+ * Publish Documents
+ *
+ * Publishes the documents to Horizon. This is done asynchronously
+ * as we're not interested in the response
+ *
+ * @param log
+ * @param {{id: string, type: string}[]} documents
+ * @param {string} serviceId
+ * @param {string} horizonId
+ * @returns {Promise<void>}
```

Comment:
I'm not sure this signature is correct.

The function doesn't return anything, so potentially just `void` ? 

Maybe wrong on this.

Reply:
I'm with you. Removed entirely.